docs(node pools): updates to node pools content to reflect move to beta (#9460)
strimzi/strimzi-kafka-operator: documentation/modules/configuring/proc-migrating-clusters-node-pools.adoc

// assembly-config.adoc

[id='proc-migrating-clusters-node-pools-{context}']
= Migrating existing Kafka clusters to use Kafka node pools

[role="_abstract"]
This procedure describes how to migrate existing Kafka clusters to use Kafka node pools.
After you have updated the Kafka cluster, you can use the node pools to manage the configuration of nodes within each pool.

> NOTE: While the `KafkaNodePools` feature gate that enables node pools is in beta phase, replica and storage configuration in the `KafkaNodePool` resource must also be present in the `Kafka` resource. The configuration is ignored when node pools are being used. 

Reviewer: Same as above ... maybe this would be better?

```suggestion
NOTE: Currently, replica and storage configuration in the `KafkaNodePool` resource must also be present in the `Kafka` resource. The configuration is ignored when node pools are being used. 
```
